pkgs/by-name/hn/hn-text/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "hn-text";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "piqoni";
+    repo = "hn-text";
+    rev = "v${version}";
+    hash = "sha256-4yf3yZYvJtpYwTYuUIAYM9Rvm2FhUhTePYoTYn0pv6c=";
+  };
+
+  vendorHash = "sha256-Y77oNc6z576Gex1lgrvNybJBxwRd0/Gtc+EiQqBikas=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "A text-first HN client";
```

The preference is to avoid use of indeterminate articles at the beginning of the description and use proper capitalisation. You could also use the first sentence from the Readme instead. That way the description contains the word hacker and news, which makes the package more findable.

```suggestion
    description = "Fast, easy-to-use and distraction-free Hacker News terminal client";
```